### DISCUSSION TITLE
Web/Web/ Error ?

### DISCUSSION THREAD

**Discussion by PatH007** [LibreBooking/librebooking#516] [Q&A] [answered]
Hi there.
I just start to install librebooking on my server via FTP.
I tried to adapt config.php and tried https://mywebsite.fr/librebooking/Web, but it automatically goes back to https://mywebsite.fr/librebooking/Web/Web with an 404 error...
No idea?
Is there any relationship with session.autostart ?
Possible problem with .htaccess provided ?

```
RewriteCond %{REQUEST_URI} !^/Web/
RewriteRule ^(.*)$ /Web/$1 [R]
```

thanks for any help.

**Comment by PatH007** [comment]
I tried to solve that changing htaccess lines.
RewriteCond %{REQUEST_URI} !^/librebooking/Web/
RewriteRule ^(.*)$ /librebooking/Web/$1 [R]

**Comment by bpinsard** [comment]
This is also true when logging out on a functional instance (or auto-logging-out after inactivity). It sends to that erroneous URL.

**Comment by PierreSerpe** [comment]
I get the same script.url problem ending with /Web/Web in the link sent through automatic emails which are generated after a password reset or removing reservation for example.
Thanks in advance for any help

**Comment by grafch** [comment, marked as answer]
Hi there,
I set it to
```
RewriteCond %{REQUEST_URI} !^/Web/
RewriteRule ^(.*)$ /Web/$1 [L,R=301]

```
since it newer redirected to /Web/Web

**Reply by ehk20** [reply]
We're seeing the same issue but because we are working out of a sub-directory /librebooking/ (as in the original issue) this doesn't work for some reason even if I had hoped that simply updating it to:
```
RewriteCond %{REQUEST_URI} !^/librebooking/Web/
RewriteRule ^(.*)$ /librebooking/Web/$1 [L,R=301]
```

would. Any ideas?

**Reply by ehk20** [reply]
Just realising there is a better way to do this I've tried:
```
RewriteBase /librebooking/
RewriteCond %{REQUEST_URI} !^/Web/
RewriteRule ^/(.*)$ /Web/$1 [R]
```

but I still get the dreaded https://HOST/librebooking/Web/Web scenario in many instances, so I'm now inclined to think this is not happening in the top level .htaccess but somewhere else.

**Comment by PierreSerpe** [comment]
Hi thanks for the hint. However, I had to adapt slightly.
The following worked for me,(but only for the reset psw link):
```
RewriteCond %{REQUEST_URI} ^/$
RewriteRule ^(.*)$ /Web/$1 [L,R=301]
```
